docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/webhook/page.mdx

---
title: 'Webhook (HTTP)'
description: 'Receive events from external systems via HTTP webhooks and stream them into ClickHouse'
---
import { Callout } from 'nextra/components'

# Webhook (HTTP)

A webhook is an HTTP-based event delivery mechanism where external systems send real-time data to a specified endpoint. It enables push-based integrations by triggering automated data ingestion whenever events occur, without the need for polling APIs.

## Using Webhook (HTTP) with GlassFlow

GlassFlow can receive webhook payloads from any HTTP-capable producer and write the events to ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Webhook (HTTP) connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Webhook (HTTP) connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
